src/pages/TaskPage/Task.jsx

```diff
@@ -9,7 +9,7 @@ const Task = (props) =>{
   ];
   const createdDate = createdDateObj.getHours()%12 +':'+ createdDateObj.getMinutes() + (createdDateObj.getHours() > 12 ? 'PM' : 'PM') +' '+ monthNames[createdDateObj.getMonth()] + ' ' + createdDateObj.getFullYear();
   const isCompleteClass = theTask.completed ? 'complete' : 'incomplete';
-  const flipAction = theTask.flip ? 'flip' : '';
```

Comment:
Using the completed prop directly to determine flipAction class. This will cause the task to run animation on first render if it is complete. If you don't mind this, this is the simplest way to implement the animation. Otherwise see the changes on ToDoListItem.jsx